raiden/tests/integration/api/test_restapi.py

```diff
@@ -2533,67 +2539,48 @@ def test_api_testnet_token_mint(api_server_test_instance: APIServer, token_addre
 
 
 @pytest.mark.parametrize("number_of_nodes", [2])
-def test_api_payments_with_lock_timeout(
```

Comment:
why did  you delete the old test?

Reply:
added it back, waiting for the build

Reply:
Was it removed? i think i just renamed it.